release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -2940,7 +2937,7 @@ In the following tables, features are marked with the following statuses:
 |====
 
 [discrete]
-=== Hosted Control Plane Technology Preview features
+=== Hosted control planes Technology Preview features
 
 .Hosted Control Plane Technology Preview tracker
```

Comment:
Should this be Hosted control plane as well?

Reply:
Good catch, thanks!